### ISSUE TITLE
Introduce CI

### ISSUE THREAD

**Issue by yaroslav-orel** [dmverner/board-games-fun#2]
Might be useful to have CI in place. It will allow us to have testing, test coverage, style checks static analysis on each push and pull request.

**Comment by dmytro-verner** [comment]
Good idea. By the way, Github suggests using CI and one of them is Travis CI which is free for our purposes.

**Comment by dmytro-verner** [comment]
The first build is passed in Travis CI.
